x-pack/test/security_solution_endpoint/apps/endpoint/endpoint_list.ts

```diff
@@ -85,17 +84,16 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       });
 
       it('finds data after load and polling', async () => {
-        await esArchiver.load('endpoint/metadata/api_feature', { useCreate: true });
-        await pageObjects.endpoint.waitForTableToHaveData('endpointListTable', 100000);
+        await esArchiver.load('endpoint/metadata/destination_index', { useCreate: true });
```

Comment:
Hmm @nnamdifrankie the index `metrics-endpoint.metadata_current_default` a data stream? If it is then this is fine. If it is not a data stream we should be able to omit the `{ useCreate: true }` and we'd need to put the mapping file in the directory like we used to.

If the tests work like this though, I guess that's fine too 🤷

Reply:
Well it works because we have an index template that matches the index, so you can search. I was curious about the useCreate here but I guess it did not make a difference. Ideally we should be using the transform e2e but it is expensive.

Reply:
@jonathan-buttner it's not a datastream.  My guess on why the tests work is because the index gets created by the transform that's running.  I can omit the `useCreate` to see what happens

Reply:
Na I wouldn't worry about it then. The only thing `useCreate` does is have es_archiver use `create` instead of `index` which means that it can't update a document.

Reply:
https://github.com/elastic/kibana/blob/master/packages/kbn-es-archiver/src/lib/docs/index_doc_records_stream.ts#L33-L38

If it was a data stream you'd get an error if you didn't pass in `useCreate`